mozetl/cli.py

```diff
@@ -1,17 +1,17 @@
 import click
 
+from mozetl.addon_aggregates import addon_aggregates
 from mozetl.clientsdaily import rollup as clientsdaily
 from mozetl.engagement.churn import job as churn_job
 from mozetl.engagement.churn_to_csv import job as churn_to_csv_job
 from mozetl.engagement.retention import job as retention_job
 from mozetl.experimentsdaily import rollup as experimentsdaily
 from mozetl.search import search_rollups
-from mozetl.search.aggregates import search_aggregates_click, search_clients_daily_click
+from mozetl.search.aggregates import search_aggregates_click, \
```

Comment:
Please avoid the line continuation by wrapping the imported modules in parens, i.e:
```python
from mozetl.search.aggregates import (
    search_aggregates_click, search_clients_daily_click
)
```
https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses

Reply:
Is this something we can add to flake8 rules?